src/sentry/api/endpoints/project_rule_actions.py

```diff
@@ -110,7 +111,12 @@ def execute_future_on_test_event(
                     logger.warning(
                         "%s.test_alert.unexpected_exception", callback_name, exc_info=True
                     )
-                    break
+                    error_id = sentry_sdk.capture_exception(exc)
+                    action_exceptions.append(
+                        f"An unexpected error occurred. Error ID: '{error_id}'"
```

Comment:
We kind of know at this point that this error is related to integration. Can we say something like Jira returned an unexpected error? Basically sending people's attention to their Jira instance, and maybe they can figure out the issue without filing support.

Reply:
If we encounter an error here, it may not necessarily be integration related. I've also noticed that having multiple integrations in the same alert rule also makes things confusing, so I'll try to filter this down a bit more in a follow-up PR that includes better error messaging for both of these cases.